### DISCUSSION TITLE
1.5.0

### DISCUSSION THREAD

**Discussion by chucker** [chucker/Mastonaut#82] [Announcements]
# Features

* You can now 🚀 **Boost**, ⭐️ **Favorite**, or 📕 **Bookmark** a toot from its contextual menu.
* You can view your **Favorites** or **Bookmarks** as a timeline column. #47, #53
* In addition to accounts and tags, you can now search for **Toots**. Note that, for privacy reasons, most instances won't let you arbitrarily search for just _any_ toot. Typically, things like searching by hashtag, searching for your own toots, and searching toots you've interacted with (e.g., bookmarked) will work.

# Fixes, minor changes

* Logging in to GoToSocial-based instances now works. #73
* Fixes a visual glitch (probably since macOS 11 Big Sur) in the Composing Settings. #65
* macOS Notifications were sent for even when a filter for this notification type was configured in Settings. #76
* The remaining character count didn't update correctly if there was text after a URL. #67
* **Filters:** there were multiple bugs in setting and reading filters. #77

## Contributors

@galad87 made their first contribution in this release cycle. Thanks!

<hr /><em>This discussion was created from the release <a href='https://github.com/chucker/Mastonaut/releases/tag/app-1.5.0'>1.5.0</a>.</em>

**Comment by akibasocial** [comment]
Hello.
Can we have a 1.5.x build?

**Reply by chucker** [reply]
Hey, 1.5 can be downloaded at https://github.com/chucker/Mastonaut/releases/download/app-1.5.0/Mastonaut-1.5.0.zip

Did you mean 1.6, perhaps? I’m afraid that’s still not quite polished enough.

**Reply by akibasocial** [reply]
Yeah, I meant 1.5.X like 1.5.1 or something since there's a lot of commits already on master after the 1.5 realease

**Reply by chucker** [reply]
Gotcha. I’ll try to make a new build soon.

**Reply by chucker** [reply]
Here ya go: https://github.com/chucker/Mastonaut/releases/tag/app-1.6.0

**Reply by akibasocial** [reply]
Thank you very much 😸
